packages/edit-site/src/components/global-styles/font-library-modal/font-collection.js

```diff
@@ -109,22 +116,32 @@ function FontCollection( { slug } ) {
 		[ collectionFonts, filters ]
 	);
 
+	const pageSize = Math.floor( ( window.innerHeight - 417 ) / 61 );
```

Comment:
We might need to leave a comment saying what these 417 and 61 mean.

Reply:
Comments added.